database/java/2015/5211.yaml

```diff
@@ -0,0 +1,48 @@
+cve: 2015-5211
+title: "Spring Framework: RFD Download vulnerability"
+description: >
+    Under some situations, the Spring Framework is vulnerable to a Reflected File Download (RFD) attack.
+    The attack involves a malicious user crafting a URL with a batch script extension that results in the
+    response being downloaded rather than rendered and also includes some input reflected in the response.
+cvss_v2: 4.0
+references:
+    - http://pivotal.io/security/cve-2015-5211
+    - https://access.redhat.com/security/cve/cve-2015-5211
```

Comment:
Maybe we should use link to http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2015-5211 instead (addition) to RH tracker?